lib/models/device.ts

```diff
@@ -1695,9 +1801,11 @@ const getDeviceModel = function (deps, opts) {
 		 * 	// Raspberry Pi
 		 * });
 		 */
-		getDisplayName: async (deviceTypeSlug) => {
+		getDisplayName: async (
+			deviceTypeName: string,
+		): Promise<string | undefined> => {
```

Comment:
:grey_exclamation: `getManifestBySlug` throws instead of returning `undefined.
```suggestion
		): Promise<string> => {
```

Reply:
@thgreasi in the catch:
```ts
				if (error instanceof errors.BalenaInvalidDeviceType) {
					return;
				}